packages/components/src/components/as-category-widget/as-category-widget.tsx

```diff
@@ -204,11 +212,13 @@ export class CategoryWidget {
       'as-category-widget__category--selected': isSelected
     };
 
+    const displayValue = this.formatValue ? this.formatValue(category.value) : readableNumber(category.value);
```

Comment:
This can be missleading, should we pass `readableNumber` to formatValue ? 

- An user see `10` dy default 
- The user creates a format function `number => ${number}€`
- The user will see: `10.00000000€`

Reply:
I thought about it the other day and I couldn't come up with the right thing. Some developers would want to have the whole unparsed number and some other might want to have the readable number.

What about returning both values in an object?